templates/Gemfile.erb

```diff
@@ -12,6 +12,7 @@ gem "oj", "3.5.1"
 <% if !is_v1 %>
 gem "fluent-plugin-secure-forward"
 gem "fluent-plugin-record-reformer"
+gem "fluent-plugin-rewrite-tag-filter", "1.6.0"
```

Comment:
Some questions?
a) You shouldn't need to specify a version number here ... the dependencies are specified correctly on the gem so bundler will resolve the latest compatible version for you?
b) why are you not including the plugin for the v1 based images?

Reply:
a) v1 and v0.12 uses a different version of the rewrite-plugin-filter - https://github.com/fluent/fluent-plugin-rewrite-tag-filter
b) Including them now